kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
```

Comment:
[Importance 2/10]

Maybe we should create a namespace `namespace: website` and have the namespace created in another file.

Having some namespace makes it easy to say for instance thatpod inside  `website` should have a service mesh attached on.

Reply:
I did a partial step - I moved everything out of default namespace but I didn't go with the complication of having different namespaces for different groups of pods.